index.d.ts

```diff
@@ -813,3 +827,15 @@ export namespace Tours {
 	export type ToursQueryDirection = 'asc' | 'desc';
 	export type ToursQuerySortBy = 'price' | 'rating' | 'top_sellers' ;
 }
+
+export namespace Collection {
+	export interface Collection {
+		id: number;
+		nameLong: string;
+		nameShort: string | null;
+		description: string | null;
+		tags: Places.Tag[];
+		places: Promise<Places.Place[]>;
```

Comment:
This should be without `Promise`, just `Places.Place[]`

Reply:
fixed